resqpy/property.py

```diff
@@ -2724,8 +2725,8 @@ def add_cached_array_to_imported_list(self,
             if not discrete and np.all(np.isnan(zorro)):
                 min_value = max_value = None
             else:
-                min_value = np.nanmin(zorro)
-                max_value = np.nanmax(zorro)
+                min_value = int(np.nanmin(zorro))
```

Comment:
The logic is not quite right here.  The int() must only be applied when discrete is True.

Reply:
I think that will do it, so I'll mark this as resolved.  Could I suggest a test with a discrete array, with a null value defined (typically -1) and all the values in the array set to the null value.  Ideally the min and max values would end up as None in that scenario.  If it comes out different, we might need to strip out the int(np.nanmin()) and use a simple np.min() on the masked zorro array.